backend/Models/Data/Sighting.cs

```diff
@@ -0,0 +1,21 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+
+namespace WhaleSpotting.Models.Data;
+
+public class Sighting
+{ 
+    [Key]
+    public int Id { get; set; }
+    public int UserId { get; set; }
+    public int WhaleSpeciesId { get; set; }
+    public float Lattitude { get; set; }
+    public float Longitude { get; set; }
+    public int AreaNameId { get; set; }
+    public string PhotoUrl { get; set; }
+    public string Description { get; set; }
+    public DateTime DateTime { get; set; }
+    public bool ApprovalStatus { get; set; }
```

Comment:
usually a boolean field that represents whether something has a particular state/value or not would be named something like `IsValue` so in this case it would be `IsApproved` this will make it easier for the meaning of the field to be read when it's being used in the codebase - please can you make that change for this?
Also I think this field should have the default value of false whenever a sighting is created. Please can you add that as well